test/test.js

```diff
@@ -25,24 +25,6 @@ describe('nodejs-docker', () => {
       }, 3000);
     });
   });
-
-  it('can use a custom nodejs version', (done) => {
```

Comment:
In that case, can you keep this tests around. We do want travis to continue running this test as our CI for now.